### PR TITLE
Added docs to readme (and small fixes)

### PR DESCRIPTION
**Pull request by KobeW50**
closes https://github.com/ReVanced/revanced-manager/issues/962

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,20 @@
 The official ReVanced Manager based on Flutter.
 
 ## 🔽 Download
-To download latest Manager, go [here](https://github.com/revanced/revanced-manager/releases/latest) and install the provided APK file.
+To download the latest Manager, go [here](https://github.com/revanced/revanced-manager/releases/latest) and install the provided APK file.
```
Comment (oSumAtrIX) [inline README.md:6]: This should say "To download ReVanced Manager" as "ReVanced Manager" is a product name.

File: README.md [hunk 2]
```diff
@@ -3,17 +3,20 @@
...
 
 ## 📝 Prerequisites
 1. Android 8 or higher
 2. Does not work on some armv7 devices
 
+## 📃 Documentation
+The documentation can be viewed [here](https://github.com/revanced/revanced-manager/tree/main/docs).
```
Comment (oSumAtrIX) [inline README.md:13]:
```suggestion
The documentation can be found [here](https://github.com/revanced/revanced-manager/tree/main/docs).
```

File: .github/ISSUE_TEMPLATE/bug-issue.yml [hunk 3]
```diff
@@ -105,16 +105,16 @@ body:
     validations:
       required: false
   - type: checkboxes
-    id: acknowledgements
+    id: acknowledgments
```
Comment (oSumAtrIX) [inline .github/ISSUE_TEMPLATE/bug-issue.yml:108]: `acknowledgements` is fine.

```suggestion
    id: acknowledgements

```

File: .github/ISSUE_TEMPLATE/bug-issue.yml [hunk 4]
```diff
@@ -29,7 +29,7 @@ body:
   - type: textarea
     attributes:
       label: Steps to reproduce
-      description: Add the steps to reproduce this bug including your environment.
+      description: Add the steps to reproduce this bug, including your environment.
```
Comment (oSumAtrIX) [inline .github/ISSUE_TEMPLATE/bug-issue.yml:32]:
```suggestion
      description: Add the steps to reproduce this bug, including details to your environment.

```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Comment by oSumAtrIX** [comment]
Why was this closed?

**Comment by KobeW50** [comment]
Bec i have no idea what im doing so those issue templates were still my old commits. I forgot to sync before making the pr like an idiot. Ill make a new one soon

**Pull request closed without merging**